src/app/admin/dashboard/page.tsx

```
@@ -486,21 +506,29 @@ export default function AdminDashboard() {
             </div>
             <div className="flex items-center gap-3">
               <button
-                onClick={() => router.push("/admin/visitors")}
+                onClick={() => setShowCommandPanel(!showCommandPanel)}
+                className="bg-white/20 hover:bg-white/30 p-2 sm:px-4 sm:py-2 rounded-lg transition-colors flex items-center gap-2 flex-shrink-0"
+                title="Command Execution"
+              >
+                <Terminal className="w-4 h-4 sm:w-5 sm:h-5" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)